lib/slack/real_time/client.rb

```diff
@@ -11,12 +11,6 @@ class ClientAlreadyStartedError < StandardError; end
       include Api::Message
       include Api::Typing
 
-      @events = {}
-
-      class << self
-        attr_accessor :events
-      end
```

Comment:
This was left unused when event hooks were migrated to inside the stores (https://github.com/slack-ruby/slack-ruby-client/commit/d612c35ee7b8f59a68a35f40a20fc3c296c77cd1)